optaplanner-core/src/main/java/org/optaplanner/core/impl/partitionedsearch/DefaultPartitionedSearchPhase.java

```diff
@@ -48,7 +47,7 @@
  * @param <Solution_> the solution type, the class with the {@link PlanningSolution} annotation
  */
 public class DefaultPartitionedSearchPhase<Solution_> extends AbstractPhase<Solution_>
-        implements PartitionedSearchPhase<Solution_>, PartitionedSearchPhaseLifecycleListener<Solution_> {
+        implements PartitionedSearchPhase<Solution_> {
```

Comment:
If we remove PartitionedSearchPhaseLifecycleListener, then it still needs to implement SolverLifecycleListener.